### ISSUE TITLE
Aspire Project with Tests, Compile Error when Solution is name with  "." (something like "Aspire.Sample")

### ISSUE THREAD

**Issue by srinivasan-mk** [dotnet/aspire#4819]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Create a aspire solution with Tests project and name the solution with "." in it (something like Aspire.Sample)

Naviagate to Tests Project and WebTests.cs, where we can find an error Or Build the solution to find out the error.

Project Template Generated code :
`var appHost = await DistributedApplicationTestingBuilder.CreateAsync<Projects.Aspire.Sample_AppHost>();`

Workaround:
Projects.Aspire.Sample_AppHost should be renamed as Projects.Aspire_Sample_AppHost

### Expected Behavior

Solution should be built successfully

### Steps To Reproduce

[Aspire.Sample.zip](https://github.com/user-attachments/files/16139199/Aspire.Sample.zip)

### .NET Version info

.NET SDK:
 Version:           8.0.302
 Commit:            ef14e02af8
 Workload version:  8.0.300-manifests.5273bb1c
 MSBuild version:   17.10.4+10fbfbf2e

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.26100
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\8.0.302\

### Anything else?

![image](https://github.com/dotnet/aspire/assets/13333003/d8e97e0a-bfde-4323-966c-2e68811dc305)

**Comment by davidfowl** [comment]
@eerhardt did we fix this?

**Comment by eerhardt** [comment]
This is fixed in `8.2`. If I take the sample app, update it to `<PackageReference Include="Aspire.Hosting.AppHost" Version="8.2.0" />`. Then change the code in `WebTests.cs` to:

```
        var appHost = await DistributedApplicationTestingBuilder.CreateAsync<Projects.Aspire_Sample_AppHost>();
```

It builds successfully.

Closing.
